metadata/process_metadata.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
the name of this file could be improved. What type of metadata are you processing? What do you mean by "process"?

Perhaps `metadata/update_platemaps_and_barcodes_to_add_pathway_info.ipynb` is a better (albeit long) name?

Reply:
I updated it to update_platemap_and_barcodes_with_pathway_info.ipynb.